src/components/search/advancedSearch/popover/Popover.module.css

```diff
@@ -11,10 +11,13 @@
   z-index: 3;
   margin-top: 150px;
   box-shadow: -1px 20px 20px -10px rgba(0, 0, 0, 0.06);
-  border-bottom: 1px solid var(--iron);
   background: var(--concrete);
 }
 
+.popoverContainer[aria-expanded="true"] {
+  border-bottom: 1px solid var(--iron);
+}
+
```

Comment:
Og dette er ikke nok alene? Altså uden z-index?

Reply:
Man kunne måske tilføje noget a la:

```
.popoverContainer:not([aria-expanded="true"]) {
  border: none;
}
```

Hvis det giver mening? Så vi slipper for at rode mere med Stacking-context / z-index :sweat_smile: